src/matrix_cookbook/for_mathlib/ring.inverse.lean

```diff
@@ -0,0 +1,85 @@
+import data.matrix.basic
+import linear_algebra.matrix.nonsingular_inverse
+
+variables {R: Type*}[division_ring R][group R]
+variables (A B C : R)
+
+lemma eq_161 {hA: A ≠ 0}{hAadd1: A + 1 ≠ 0}: 
+  (1 + A⁻¹)⁻¹ = A*(A + 1)⁻¹ := begin
```

Comment:
I meant
```suggestion
  ring.inverse (1 + ring.inverse A) = A*ring.inverse (A + 1) := begin
```
and assuming `[ring R]` (after all, a matrix is not a `division_ring`!)

Reply:
Where can I get associativity?

Reply:
> `ring` gives you associativity
OK.
Sorry I wanted to edit that message but seems I messed it up somehow